geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/InternalLocatorClusterManagementServiceIntegrationTest.java

```diff
@@ -56,13 +57,38 @@ public void tearDown() {
     }
   }
 
+  @Before
+  public void setup() throws URISyntaxException {
+    distributionConfig = mock(DistributionConfigImpl.class);
+    cache = mock(InternalCacheForClientAccess.class);
+    managementService = mock(BaseManagementService.class);
```

Comment:
With these changes does this test need to become a unit test rather than an integration test?